cadc-tomcat/src/cadc-tomcat-start

```diff
@@ -14,6 +14,11 @@ if [ -f $CONFDIR/catalina.properties ]; then
     cat $CONFDIR/catalina.properties >> /etc/tomcat/catalina.properties
 fi
 
+if [ -f $CONFDIR/tomcat_conf.properties ]; then
```

Comment:
I would not give this a .properties extension since it is not a properties file... it's really a shell script setting up environment... just tomcat.conf is consistent with the append with catalina.properties

Also, you need to document this option in the README.md